test/integration/users.test.js

```diff
@@ -2646,4 +2652,42 @@ describe("Users", function () {
       expect(res).to.have.status(401);
     });
   });
+
+  describe("GET /users/departed-users", function () {
```

Comment:
test missing for error response? @VinuB-Dev

Reply:
As discussed, I’ve tried stubbing both the service and model layers, but the failure state doesn’t get triggered as expected. Have added a message in discord to check if there is any other way to achieve this.
Will update here if I get a solution.